security/security.manager.ts

```diff
@@ -0,0 +1,111 @@
+import { HttpBadRequestError } from '@errors/http';
+import { getEnv } from '@helper/environment';
+import { IconikToken, IconikTokenModel, IconikTokenSchema } from '@models/DynamoDB/iconik-token.model';
+import { IconikService } from '@workflowwin/iconik-api';
+import { MetadataFieldSchema } from '@workflowwin/iconik-api/dist/src/metadata/metadata-methods';
+import { CustomActionSchema } from '@workflowwin/iconik-api/src/assets/assets-methods';
+import { WebhookResponseSchema } from '@workflowwin/iconik-api/src/notifications/notifications-methods';
+import { ScanResponse } from 'dynamoose/dist/DocumentRetriever';
+import { SecurityService } from './security.service';
+
+const refreshHoursField: MetadataFieldSchema = {
+  field_type: 'integer',
+  name: 'Set Token Refresh Time',
+  description: 'WIN Automation View',
+  required: true,
+};
+
+export class SecurityManager {
+  private readonly service: SecurityService;
+
+  constructor() {
+    this.service = new SecurityService();
+  }
+
+  async initialization(iconikService: IconikService) {

```

Comment:
What if a user call this twise or more times?

Reply:
The second time this method will throw an error and will not create any resources.

Reply:
Do you think this is ok for a user? What if someone will add/move creation CA call before the MDF creation call?